kotekan/kotekan.cpp

```diff
@@ -499,9 +499,9 @@ int main(int argc, char** argv) {
         }
     }
 
-    INFO("kotekan shutdown successfully.");
+    INFO("kotekan shutdown with status: %s\n%s", get_exit_code_string(get_exit_code()), get_error_message());
```

Comment:
The syslog version of this message won't include the newline. 
```suggestion
    INFO("kotekan shutdown with status: %s", get_exit_code_string(get_exit_code()));
    If (string(get_error_message()) != "not set") {
        INFO("Fatal error message was: %s", get_error_message());
    } 
    
```